permission_handler/lib/permission_handler.dart

```diff
@@ -46,7 +46,8 @@ extension PermissionActions on Permission {
   ///
   /// Returns the new [PermissionStatus].
   Future<PermissionStatus> request() async {
-    return (await [this].request())[this];
+    final permissionStatus = (await [this].request())[this];
+    return permissionStatus!;
```

Comment:
Is the bang (!) operator really necessary here? I would expect that the result from the `(await [this].request())[this]` call should never be null (I don't see any nullable object "?" notations).

If the `permissionStatus` variable is allowed to be null we might need to investigate and solve that instead of simply adding a bang operator (as this might still cause null exceptions on runtime since we cannot be sure it's always not null).